src/components/Slider/Slider.scss

```diff
@@ -0,0 +1,54 @@
+@use '../variables';
+@use './sliderVariables';
+@use '../../../styles/mixins';
+
+$block: '.#{variables.$ns-new}slider';
+
+#{$block} {
+    display: flex;
+    flex-direction: column;
+
+    &__error {
+        &_size {
+            &_s {
+                margin-top: 0px;
+            }
+            &_m {
+                margin-top: 2px;
+            }
+            &_l {
+                margin-top: 4px;
+            }
+            &_xl {
+                margin-top: 6px;
+            }
+        }
+    }
+
+    &__top {
+        //6px is padding inside tooltip card
+        height: calc(
+            var(--g-text-body-1-line-height) + 6px + #{sliderVariables.$slider-tooltip-offset}
+        );
+        display: none;
+        &_withTooltip {
+            display: block;
+        }
+        &_size_xl {
+            height: calc(
+                var(--g-text-body-2-line-height) + 6px + #{sliderVariables.$slider-tooltip-offset}
+            );
+        }
+    }
+
+    &__error {
+        @include mixins.text-body-1;
+        color: var(--g-color-text-danger);
+    }
+
+    &__error {
+        &_size_xl {
+            @include mixins.text-body-2;
```

Comment:
We have one size in inputs, slider should follow the same rules

Reply:
You mean that error always should have text-body-1? But then it will not correspond to design. Is it OK?

Reply:
https://github.com/gravity-ui/uikit/pull/1243/commits/2766e9b53e8b43b08e3af2c96700330ec4c8b814

Reply:
Yes, it should corelate with other control's errors